app/controllers/meetings_controller.rb

```diff
@@ -142,18 +142,24 @@ def find_meeting
   end
 
   def convert_params
-    start_date = params[:meeting].delete(:start_date)
-    start_time_hour = params[:meeting].delete(:"start_time_hour")
+    start_date = meeting_params.delete(:start_date)
+    start_time_hour = meeting_params.delete(:"start_time_hour")
     begin
       timestring = "#{start_date} #{start_time_hour}"
       time = Time.zone.parse(timestring)
-      params[:meeting][:start_time] = time
+      meeting_params[:start_time] = time
     rescue ArgumentError
-      params[:meeting][:start_time] = nil
+      meeting_params[:start_time] = nil
     end
-    params[:meeting][:duration] = params[:meeting][:duration].to_hours
+    meeting_params[:duration] = meeting_params[:duration].to_hours
     # Force defaults on participants
-    params[:meeting][:participants_attributes] ||= {}
-    params[:meeting][:participants_attributes].each { |p| p.reverse_merge! attended: false, invited: false }
+    meeting_params[:participants_attributes] ||= {}
+    meeting_params[:participants_attributes].each { |p| p.reverse_merge! attended: false, invited: false }
+  end
+
+private
```

Comment:
Indent access modifiers like `private`.<br>Keep a blank line before and after `private`.